datastore/elastic_search/query.py

```diff
@@ -185,16 +223,31 @@ def add_data_elastic_search(
             'variants': record.get('variants'),
         }
         str_query.append(query_dict)
-        if len(str_query) > ELASTICSEARCH_BULK_HELPER_MESSAGE_SIZE:
+        if len(str_query) > constants.ELASTICSEARCH_BULK_HELPER_MESSAGE_SIZE:
             helpers.bulk(connection, str_query, stats_only=True, **kwargs)
             str_query = []
 
     if str_query:
         helpers.bulk(connection, str_query, stats_only=True, **kwargs)
 
 
-def dictionary_unique_words(connection, index_name, doc_type, entity_name, **kwargs):
+def get_entity_unique_values(
+        connection, index_name, doc_type, entity_name,
+        value_search_term=None, variant_search_term=None, empty_variants_only=False,
+        **kwargs):
     """
+    Search for values in entity with filters
+
+    Args:
+        connection: Elasticsearch client object
+        index_name: The name of the index
+        doc_type: The type of the documents that will be indexed
+        entity_name (str): name of the entity for which the data is to be fetched
+        value_search_term (str): Filter values with the specific search term
+        variant_search_term (str): Filter variants with the specific search term
+        empty_variants_only (bool): Search for values with empty variants only
+    Returns:
+        values (list): List of values which match the filters and search criteria
```

Comment:
Minor, 
no need to mention any name on left side in `Returns` section. Only type is enough

```
list: List of values which match the filters and search criteria
```